frontend/src/Navigation/myKingdom.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
great idea for creating a basic 'layout', we can use this later!
but please don't put it into the navigation folder, you didn't put the Settings page here either!
create a folder for it, like MainPage, or something like that.